### DISCUSSION TITLE
Contribution Guide

### DISCUSSION THREAD

**Discussion by hkbertoson** [prismaneui/prismane#15] [Ideas]
I was wanting to help contribute to this project. But there does not seem to be a contribution guide. I forked the project and it took some work to actually get it running as well.

**Comment by spleafy** [comment]
Hello,
Thank you for noting this issue, we will create a contribution guide as soon as possible

**Comment by spleafy** [comment]
Created a CONTRIBUTING.md file, you can look at it and let me know if anything should be changed. Thanks for the feedback!
